### PR TITLE
Change a snippet in the Display Density topic

### PR DESCRIPTION
**Pull request by mddragnev**

### CODE REVIEW

**Comment by mddragnev** [comment]
@HUSSAR-mtrela Why are u setting in-development label?

File: doc/en/components/grids/_shared/display-density.md [hunk 1]
```diff
@@ -756,12 +737,16 @@ We can now extend our sample and add `RowHeight` property to the `{ComponentName
...
+             RowHeight="rowHeight"
+             DisplayDensity="@density">
 </{ComponentSelector}>
+
+@code {
+    private string rowHeight = "80px";
+}
  ```
 
  <!-- WebComponents -->
```
Comment (dkamburov) [inline doc/en/components/grids/_shared/display-density.md:752]: The code snippet bellow is wrong. `display-density="Cosy"` is wrong, should be display-density="cosy"

Comment (dkamburov) [reply doc/en/components/grids/_shared/display-density.md:752]: And there is one more occurrence of it in this doc

**Review by dkamburov — CHANGES_REQUESTED** [review]
We need to remove property editor panel from code snippets and ComponentRenderer related code

**Review by dkamburov — APPROVED** [review]

**Pull request merged**